### ISSUE TITLE
Design consistent pattern options to show, collect, and update contact information in forms

### ISSUE THREAD

**Issue by sshein** [department-of-veterans-affairs/vets-design-system-documentation#305]
Currently, there are various ways throughout VA.gov that we allow Veterans to view, gather, and update contact information.

We need to have a variety of ways to some extent because of different stakeholder use cases, but we should align as much as possible on labeling and use cases.

Jim, Jonathan, and Kristen are all kind of working on this initiative. I (Shawna) am planing on continuing the initiative.

**Comment by kristenmcconnell** [comment]
The Check-in team has been working on a pattern for reviewing, then editing contact information within a workflow/form. We have been syncing with the VA.gov Profile and Claims and Appeals teams throughout our process.

VA.gov Profile completed discovery research on this topic ([research findings link](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/contact-information/discovery/improving-editing-mid-task.md)). Within their research, they found that multiple VA tasks have this need, including
- Managing notification preferences
- Filling out a form (such as notice of disagreement or pre-check in questionnaire)
- Checking in for an appointment at a VAMC
- When scheduling an appointment 

Across numerous user research studies, check-in verified the need to review/update contact information as well as usability tested an editing pattern. 
-  Staff explained that keeping this information up-to-date was important, because it impacts where medications are sent, “patient safety,” accurate beneficiary travel calculations, wellness checks and more.   ([More about the value to staff can be reviewed in this research study.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/research/veteran-facing/mvp-UAT/research-findings.md))
- VHA Directive 1604 also exists, which specifies the review and collection of this information frequently.
  - Staff used this to help explain why a review of contact information is currently completed during kiosk and in-person check-ins.
- Veterans also want to edit this information during check-in.
  - 50% of participants wanted to edit in [this research study](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/research/veteran-facing/mvp-usability/research-findings.md).
  - This was a key finding [within another study](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/research/veteran-facing/phase2-usability/research-findings.md): Veterans want to be able to update their contact information with this check-in workflow; however, they accept the current workflow with the question “Do you need to update any information?” and directions to talk to staff to complete updates.
  - We also usability tested a specific editing pattern/design [within this study](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/research/veteran-facing/pre-check-in-usability/research-findings.md). Even though the design utilized modals, the pattern tested well.  We had been syncing with the Claims and Appeals team on what the actual pattern looks like.
  
After speaking with @rtwell  and accessibility, we adjusted the design from modals to pages. We also changed the labels (such as Mailing address) on the review page to bold body copy from H3. Accessibility recommended this change, so the review section will be read together for a screenreader. The final design pattern that we are proposing is available for [review on Sketch.](https://www.sketch.com/s/5331b114-280d-4ff5-8d36-ec49b1696b9e/p/5BCAB931-5EBD-4208-BDA4-592B768DE26F)

cc @andaleliz, @kevinstachura, and @chrvalla
@caw310

**Comment by GnatalieH** [comment]
Looks cool! @kristenmcconnell are you suggesting a new Experimental Design pattern? If you'd like the Design System Council to consider your experimental pattern, [here are the steps](https://design.va.gov/documentation/contributing-to-the-design-system) for contributing to the design system. Looks like this work would be valuable. Thanks!

**Comment by kristenmcconnell** [comment]
@GnatalieH Yes, I am interested in suggesting this pattern to the experimental design system. When I was completing Step 1 of the experimental design process, I saw this ticket already existed in the proposed section of the backlog. I thought this ticket was really asking for the same new pattern (after talking with Shawna) that I now have more detail on. I was just adding my documentation to it, since it still seemed open. To confirm, I should open a new ticket and not connect it to this existing ticket?

**Comment by GnatalieH** [comment]
@kristenmcconnell I'd go ahead and create an Experimental Design request ticket because it will add the experimental-design-system-team tag. You can always link back to your comment in this ticket instead of duplicating everything. Thanks!

**Comment by kristenmcconnell** [comment]
@GnatalieH Will do. Thanks for clarifying.

**Comment by caw310** [comment]
Closing as Kristen submitted an experimental design request that addresses this. https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/619
